core-services/ci-secret-bootstrap/_config.yaml

```diff
@@ -975,6 +975,54 @@ secret_configs:
     - non_app_ci
     name: cluster-secrets-gcp
     namespace: ci
+- from:
+    gce.json:
+      attachment: credentials.json
+      bw_item: gcp-ci-provisioner-2
+    insights-live.yaml:
+      attachment: insights-live.yaml
+      bw_item: insights-ci-account
+    ops-mirror.pem:
+      attachment: cert-key.pem
+      bw_item: mirror.openshift.com
+    pull-secret:
+      dockerconfigJSON:
+      - auth_bw_attachment: auth
+        bw_item: api.ci-pull-secret
+        registry_url: registry.svc.ci.openshift.org
+      - auth_bw_attachment: token_image-puller_app.ci_reg_auth_value.txt
+        bw_item: build_farm
+        registry_url: registry.ci.openshift.org
+      - auth_bw_attachment: auth
+        bw_item: cloud.openshift.com-pull-secret
+        email_bw_field: email
+        registry_url: cloud.openshift.com
+      - auth_bw_attachment: auth
+        bw_item: quay.io-pull-secret
+        email_bw_field: email
+        registry_url: quay.io
+      - auth_bw_attachment: auth
+        bw_item: registry.connect.redhat.com-pull-secret
+        email_bw_field: email
+        registry_url: registry.connect.redhat.com
+      - auth_bw_attachment: auth
+        bw_item: registry.redhat.io-pull-secret
+        email_bw_field: email
+        registry_url: registry.redhat.io
+    ssh-privatekey:
+      attachment: ssh-privatekey
+      bw_item: gcp-ci-provisioner-2
+    ssh-publickey:
+      attachment: ssh-publickey
+      bw_item: gcp-ci-provisioner-2
+    telemeter-token:
+      bw_item: telemeter
+      field: telemeter-token
+  to:
+  - cluster_groups:
+    - non_app_ci
+    name: cluster-secrets-gcp-2
```

Comment:
Can we give it a name that is a bit more meaningful, i.E. includes the gcp project name (that isn't a secret, is it?)

Reply:
Well, I think that this name is as meaningful as it can get. For example, the first one `cluster-secrets-gcp` doesn't include the GCP project name...

Reply:
In addition, this secret contains data that doesn't have to do with any of the GCP accounts

Reply:
Well the first one was unique by the fact that it is for gcp. Now that we have a second one, the uniqueness by being for gcp is gone. Can we please include the gcp project name in the secret name?

Reply:
I added the service account's name

Reply:
updated

Reply:
Where? I only see `gcp-ci-provisioner-2` that is not the project name, is it?

Reply:
It's the service account name.

Reply:
What does the service account name help with? The main difference of this credential to the other crtedential is that this is for a different project or not? And ServiceAccountNames in turn are scoped to a project, so knowing the SA name without the project name doesn't really help a lot?

Reply:
updated